wgpu-types/src/lib.rs

```diff
@@ -662,8 +702,10 @@ bitflags::bitflags! {
         const COMPARISON_SAMPLERS = 0x0000_0100;
         /// Supports different blending modes per color target.
         const INDEPENDENT_BLENDING = 0x0000_0200;
-        /// Supports samplers with anisotropic filtering
-        const ANISOTROPIC_FILTERING = 0x0001_0000;
+        /// Supports samplers with anisotropic filtering.
+        const ANISOTROPIC_FILTERING = 0x0000_0400;
```

Comment:
can we keep the special status for anisotropic? it's the only feature that's not required.
So maybe have it always at the end, and with a value distinct from all others?

Reply:
We totally could, but I don't see the real point considering flags are supposed to be opaque.

Reply:
some note that it is not required to be webgpu conform would be nice though - given that the flags are documented as `Binary flags listing various ways the underlying platform does not conform to the WebGPU standard.` it looks now like it would be required for conforimity